src/main/java/tetris/KeyBindings.java

```diff
@@ -8,7 +8,7 @@
 import javafx.scene.input.KeyCode;
 
 public final class KeyBindings implements Iterable<Entry<KeyCode, String>> {
-    private HashMap<KeyCode, String> bindings = new HashMap<KeyCode, String>();
+    private HashMap<KeyCode, String> bindings = new HashMap<>();
```

Comment:
I don't understand. Why would you leave out the type specifiers and do an implicit typecast? This strikes me as abuse of the generics framework.